src/three-components/SmoothControls.ts

```diff
@@ -173,16 +188,25 @@ export interface ChangeEvent extends Event {
  * ensure that the camera's matrixWorld is in sync before using SmoothControls.
  */
 export class SmoothControls extends EventDispatcher {
+  protected[$idealCameraDistance]: number = -1;
+
   private[$interactionEnabled]: boolean = false;
 
   private[$options]: SmoothControlsOptions;
   private[$upQuaternion]: Quaternion = new Quaternion();
   private[$upQuaternionInverse]: Quaternion = new Quaternion();
-  private[$spherical]: Spherical = new Spherical();
-  private[$targetSpherical]: Spherical = new Spherical();
-  private[$previousPosition]: Vector3 = new Vector3();
   private[$isUserChange]: boolean = false;
 
+  private[$spherical]: Spherical = new Spherical();
+  private[$destSpherical]: Spherical = new Spherical();
+  private[$thetaDamper]: Damper = new Damper();
+  private[$phiDamper]: Damper = new Damper();
+  private[$radiusDamper]: Damper = new Damper();
```

Comment:
We are completely inconsistent with this, but I recently learned that TypeScript will infer a class field or variable's type based on initial assignment. Feel free to take advantage of this for brevity whenever you feel like it 💁‍♂